luthier/config/src/tables.lua

```diff
@@ -76,15 +78,21 @@ local air_quality_dimensions = {
 
 M.physical = {
     wikiticker = {
+        dateTimeZone = "UTC",
+        type = "strict",
         metrics = {
             "added",
             "delta",
             "deleted"
         },
         dimensions = wikipedia_dimensions,
-        granularity = "hour"
+        granularity = "hour",
+        logicalToPhysicalColumnNames = {
```

Comment:
Should document this key in the documentation above.

Reply:
done.